android-gif-drawable/src/main/c/drawing.c

```diff
@@ -18,6 +18,9 @@ static inline void blitNormal(argb *bm, GifInfo *info, SavedImage *frame, ColorM
 	const int_fast16_t transpIndex = info->controlBlock[info->currentIndex].TransparentColor;
 	const GifWord frameWidth = frame->ImageDesc.Width;
 	const GifWord padding = info->stride - frameWidth;
+	if (info->rasterSize < frame->ImageDesc.Height * frame->ImageDesc.Width) {
+		return;
```

Comment:
I think we should set Error field here, but currently there is no appropriate value. However, IMAGE_NOT_CONFINED used to exist. I'll add it in next PR.